src/components/Tooltip/index.js

```diff
@@ -70,6 +70,8 @@ const Tooltip = props => {
         ? props.renderTrigger(props)
         : props.renderTrigger
 
+    if (props.disableTooltip) return props.renderTrigger
```

Comment:
Should this be `tooltipTrigger` given that there's a check if it's a function right above?

Reply:
yes, it should! Good catch.